assets/js/stripe-payment-request.js

```diff
@@ -640,7 +640,7 @@ jQuery( function( $ ) {
 				$( '#wc-stripe-payment-request-wrapper, #wc-stripe-payment-request-button-separator' ).show();
 			} else if ( wc_stripe_payment_request.isBrandedPaymentRequestButton( prButton ) ) {
 				$( '#wc-stripe-payment-request-wrapper, #wc-stripe-payment-request-button-separator' ).show();
-				$( '#wc-stripe-payment-request-button' ).append( prButton );
+				$( '#wc-stripe-payment-request-button' ).html( prButton );
```

Comment:
Looks good. I have just one question - from what I can see there can only ever be one button displayed. Is that assumption correct? If so, I think this is good to go.

Reply:
Thanks for the review @v18! Yes, that's right. If there are multiple payment options, they can be selected in the Payment Request API modal once the button is clicked.